.travis.yml

```diff
@@ -4,25 +4,28 @@ python:
 services:
 - docker
 install:
-- pip install --no-cache-dir ruamel.yaml
```

Comment:
For my education: do you know what this used to do/was good for?

Reply:
I know you deleted it, but for completeness:

This package is used in `build.py` for rewriting `values.yaml`. It has been moved to `dev-requirements`, along with the testing dependencies.